source/features/useful-not-found-page.tsx

```diff
@@ -36,55 +38,112 @@ function parseCurrentURL(): string[] {
 	return parts;
 }
 
-// If the resource was deleted, link to the commit history
-async function addCommitHistoryLink(bar: Element): Promise<void> {
-	const parts = parseCurrentURL();
-	parts[2] = 'commits';
-	const url = '/' + parts.join('/');
-	if (await is404(location.origin + url)) {
+async function getLatestChangeToFile(): Promise<Record<string, any> | void> {
+	const url = new GitHubURL(location.href);
+	const currentBranch = url.branch;
+	const {filePath} = url;
```

Comment:
```suggestion
	const {branch: currentBranch, filePath} = url;
```

or even just this, they're all "current"
```suggestion
	const {branch, filePath} = url;
```